mmdet3d/ops/group_points/group_points.py

```diff
@@ -98,22 +98,23 @@ def forward(self, points_xyz, center_xyz, features=None):
         xyz_trans = points_xyz.transpose(1, 2).contiguous()
         # (B, 3, npoint, sample_num)
         grouped_xyz = grouping_operation(xyz_trans, idx)
-        grouped_xyz -= center_xyz.transpose(1, 2).unsqueeze(-1)
+        grouped_xyz_diff = grouped_xyz - \
```

Comment:
After modification, `grouped_xyz` stands for the grouped points coordinates before centered (not subtracting the grouping center coordinates), while `grouped_xyz_diff` is the centered grouped point coordinates.

In the following code, `new_features` is not affected, but the returned `grouped_xyz` is different. I have checked mmdet3d's codebase and `self.return_grouped_xyz` is always `False` so this won't introduce a BC-breaking at least in mmdet3d. Do you think I need to add a compatibility docs for it?

Reply:
Yes, I think we can add a brief one in this PR.

Reply:
I have updated the compatibility doc about this modification.